src/NServiceBus.RabbitMQ/RabbitMQTransportInfrastructure.cs

```diff
@@ -62,7 +86,7 @@ public override TransportSendInfrastructure ConfigureSendInfrastructure()
         {
             return new TransportSendInfrastructure(
                 () => new MessageDispatcher(channelProvider),
-                () => Task.FromResult(StartupCheckResult.Success));
+                () => Task.FromResult(DelayInfrastructure.CheckForInvalidSetting(routingTopology is ISupportNativeDelays, disableTimeoutManager)));
```

Comment:
```c#
routingTopology is ISupportNativeDelays
```

This Boolean is calculated in the constructor. Is it worth caching it in a field?